contributing.md

```diff
@@ -381,10 +381,22 @@ Merging strategy:
 
 Porting changes across branches:
 
-* Pull requests should target the lowest possible
-  [supported version](releases) where the
+* Pull requests should target the latest [supported version](releases) where the
   changes can be added in a backwards-compatible way (no API / ABI / behavior
   break in released branches).
+* To backport a pull request (i.e. from a higher version to a lower version),
+  use `git cherry-pick` instead of `git merge`, for example:
+
+        git checkout gz-<library>N
+        git pull
+        git checkout gz-<library>M
+        git pull
+        git checkout -b N_to_M_<date>
+        git cherry-pick <commits from verrsion N>
+        # Fix conflicts
+        git commit -sam"Backport from N to M"
+        # Open pull request
+        # Do not squash, rebase instead
 * Periodically, a maintainer will **forward-port** changes to newer release
```

Comment:
What about here? Isn't it now bidirectional, or mostly back-port way?

Reply:
updated sentence so that forward port does not sound like a periodic thing we do [4724943](https://github.com/gazebosim/docs/pull/525/commits/472494320b1ed0512fa384177ffa0e0377010ef8)